e2e_samples/temperature_events/LoadTesting/azure-pipeline.yml

```
@@ -0,0 +1,44 @@
+
+trigger:
+  - none
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+  DeviceCount: 2
+  MessageCount: 5
+  Interval: 10
+  ContainerCount: 2
+  evhResourceGroup: 'temperature'
+  evhNamespace: 'temperature-dev'
+  evhName: 'temperature-processing'
```

2 EventHubs globally can't have the same name. 

The `evName` is probably fine. but `evhResourceGroup` & `evhNamespace` might need to be variables, using what the Terraform script pre-pended the resources with.

```suggestion
  evhResourceGroup: $(evhResourceGroup)
  evhNamespace: $(evhResourceGroup)
  evhName: $(evhName)
```